bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="ui:chart">
+		<parameter name="provider" type="text" required="false">
+			<label>Provider</label>
+			<description>Defines the chart provider to use.</description>
```

Comment:
I wonder if we should have that displayed in the UI at all. There currently usually is only a single provider anyhow and it is rather a choice that is done by the solution and not so much by the user. We also do not have any mechanism to display available options to the user, so I would vote to leave that out (or at least to mark it advanced).

Reply:
Ok I will suppress it.